lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -207,6 +219,7 @@ class RadarChartData extends BaseChartData with EquatableMixin {
         gridBorderData: BorderSide.lerp(a.gridBorderData, b.gridBorderData, t),
         radarBorderData:
             BorderSide.lerp(a.radarBorderData, b.radarBorderData, t),
+        radarShape: t.round() == 0 ? a.radarShape : b.radarShape,
```

Comment:
In this situation, we pass the b.value directly to prevent any state issue at the middle of lerp animation.